src/main/java/task.java

```diff
@@ -0,0 +1,112 @@
+import java.util.ArrayList;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class task {
+    private int mark;
+    private String line,description;
+    ArrayList<String> namelist=new ArrayList<>();
+    ArrayList<Character> donelist=new ArrayList<>();
+    ArrayList<Character> statelist=new ArrayList<>();
+    private static final Character MARK_AS_DONE = 'X';
+    //private static Shape[] shapes = new Shape[100];
+    //private static int shapeCount = 0;
+    public task(){
+        this(0,"null");
+    }
+    public task(int mark, String line){
+        this.mark=mark;
+        this.line=line;
+    }
+    public void getList(){
+      System.out.println("Here are the tasks in your list:");
+        for(int ay=0;ay<namelist.size();ay++){
+            System.out.println((ay+1)+". ["+statelist.get(ay)+"]"
+            +"["+donelist.get(ay)+"]"+namelist.get(ay));
+        }
+    }
+    public void setList(int mark){
```

Comment:
It would help to provide description (i.e. [javadocs style](https://se-education.org/guides/conventions/java/basic.html#comments)) on what this method is doing, as there are 2-3 lines which look complex.